scripts/apps/extension-points/directives/ExtensionPointDirective.js

```diff
@@ -15,26 +15,18 @@ import React from 'react';
  */
 ExtensionPointDirective.$inject = ['extensionPoints'];
 export function ExtensionPointDirective(extensionPoints) {
-    function propsFromData(data, scope) {
-        var json = {};
-
-        _.forEach(data, (value) => {
-            json[value] = scope.$parent.$eval(value);
+    function _buildCompoment(extension, scope) {
+        // for easy access put values from parent scope into the redux store of the extension
+        _.forEach(extension.data, (value) => {
+            extension.props.store.getState()[value] = scope.$parent.$eval(value);
```

Comment:
This implementation makes it necessary for all React components to implement the Redux architecture. This is a bit limiting in my (h) opinion. I suggest the following:
- If a `store` prop is there, use the logic that is here
- If a `store` prop is not there, use the previous logic where the values get passed directly as props (as before)